graphbolt/src/cuda/unique_and_compact_impl.cu

```diff
@@ -48,17 +66,13 @@ std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> UniqueAndCompact(
         auto dst_ids_ptr = dst_ids.data_ptr<scalar_t>();
         auto unique_dst_ids_ptr = unique_dst_ids.data_ptr<scalar_t>();
 
-        // If the given num_bits argument is not in the reasonable range,
-        // we recompute it to speedup the expensive sort operations.
-        if (num_bits <= 0 || num_bits > sizeof(scalar_t) * 8) {
-          auto max_id = thrust::reduce(
-              exec_policy, src_ids_ptr, src_ids_ptr + src_ids.size(0),
-              static_cast<scalar_t>(0), thrust::maximum<scalar_t>{});
-          max_id = thrust::reduce(
-              exec_policy, unique_dst_ids_ptr,
-              unique_dst_ids_ptr + unique_dst_ids.size(0), max_id,
-              thrust::maximum<scalar_t>{});
-          num_bits = cuda::NumberOfBits(max_id + 1);
+        // If num_bits is not given, compute it to speedup the expensive sort
+        // operations later.
```

Comment:
The num_bits assignment is not happening in this block, consider rephrase the comment?